dashboards/TDinsight.json

```diff
@@ -5501,7 +5501,7 @@
               "formatType": "Time series",
               "queryType": "SQL",
               "refId": "A",
-              "sql": "select last(mem_percent) as mem_percent from log.taosadapter_system where ts >= $from and ts <= $to",
+              "sql": "select last(mem_percent)/100 as mem_percent from log.taosadapter_system where ts >= $from and ts <= $to",
```

Comment:
does TDengine support this arithmetic express?

Reply:
```shell
taos> select last(disk_used)/100 from log.dn ;
    last(disk_used)/100    |
============================
               1.401748810 |
Query OK, 1 row(s) in set (0.003840s)

taos> select last(disk_used) from log.dn ;
   last(disk_used)    |
=======================
            140.17488 |
Query OK, 1 row(s) in set (0.002831s)

taos> exit
lbhuo@vm98:~$ taos

Welcome to the TDengine shell from Linux, Client Version:2.4.0.11
Copyright (c) 2020 by TAOS Data, Inc. All rights reserved.
```